src/discolight/augmentations/coarsedropout.py

```diff
@@ -0,0 +1,63 @@
+import random
+import math
+import numpy as np
+from discolight.params.params import Params
+from .augmentation.types import ColorAugmentation
+from .decorators.accepts_probs import accepts_probs
+
+
+@accepts_probs
+class CoarseDropout(ColorAugmentation):
+    """
+    Randomly erases a rectangular area in the given image.
+    """
+
+    def __init__(self, p, num):
+        super().__init__()
+        self.p = p
+        self.num = num
+
+    @staticmethod
+    def params():
+        return Params().add("p", "", float,
+                            0.1).add("num", "", float,
```

Comment:
This should not be `float` because you are using `num` as an argument to `range` in `augment_img`,  so it needs to be an `int` instead (I think this is what is causing the snapshot tests to fail).